pkgs/applications/virtualization/virtualbox/default.nix

```diff
@@ -265,6 +266,7 @@ in stdenv.mkDerivation {
 
   passthru = {
     inherit extensionPack; # for inclusion in profile to prevent gc
+    tests.virtualbox = nixosTests.virtualbox;
```

Comment:
This doesn't actually run the tests when running `nix-build -A virtualbox.passthru.tests` because `nixosTests.virtualbox` is an attrset.

Reply:
Good catch. I believe I fixed this in https://github.com/NixOS/nixpkgs/commit/5eb28724fe3a05a6bb9b9f5cbd44c5d73526c6a9.